test/index.test.ts

```diff
@@ -58,14 +58,13 @@ test('MTLS_ENABLED with no TLS params should error', async (t) => {
       'TLS_PRIVATE_KEY , TLS_PUBLIC_KEY and  TLS_CA environment variables are required when MTLS_ENABLED is set to true.',
     )
   }
-  // const api = await expose(adapter)
+  // Const api = await expose(adapter)
```

Comment:
Not relevant to the ticket but why don't we just remove this commented out code completely?